src/main/java/pl/confitura/jelatyna/admin/Controller.java

```diff
@@ -0,0 +1,54 @@
+package pl.confitura.jelatyna.admin;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.confitura.jelatyna.common.TokenGenerator;
+import pl.confitura.jelatyna.email.EmailService;
+
+import javax.validation.Valid;
+import java.util.List;
+
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+import static org.springframework.web.bind.annotation.RequestMethod.PUT;
+
+@RestController("adminController")
+@RequestMapping("/api/admin")
+public class Controller {
```

Comment:
1. wyszukiwanie klasy po nazwie... myślę że to dość istorne żeby płynnie poruszać się po projekcie. Ale w sumie nie wiem... dziwnie się czuje jak wyszukuję Repo.. i mi znajduje kilka klas o identycznych nazwach
2. to trzeba wyłączyć spring-data-rest - bo automatycznie mapuje nam wszystkie repozytoria (/admins też dodał)